### PR TITLE
chore: normalize spelling for PHP + version

### PR DESCRIPTION
**Pull request by kesselb**
### ☑️ Resolves

Ran into when checking the PDF document.

### CODE REVIEW

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.rst [hunk 1]
```diff
@@ -21,10 +21,10 @@ Make sure your ``appinfo/info.xml`` allows for Nextcloud 24.
 Back-end changes
 ----------------
 
-PHP8.1
+PHP 8.1
 ^^^^^^
```
Comment (nickvergessen) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.rst:25]:
```suggestion
^^^^^^^
```

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst [hunk 2]
```diff
@@ -39,20 +39,20 @@ Removed APIs
 Back-end changes
 ----------------
 
-PHP7.4
+PHP 7.4
 ^^^^^^
```
Comment (nickvergessen) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:43]:
```suggestion
^^^^^^^
```

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst [hunk 3]
```diff
@@ -39,20 +39,20 @@ Removed APIs
...
 
-In this release support for PHP7.4 was dropped. Make sure your app compatible with PHP8.0 or higher.
+In this release support for PHP 7.4 was dropped. Make sure your app compatible with PHP 8.0 or higher.
 
-PHP8.2
+PHP 8.2
 ^^^^^^
```
Comment (nickvergessen) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:48]:
```suggestion
^^^^^^^
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.rst (hunk 1 above)
Comment (kesselb) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.rst]: :see_no_evil:

**Review by nickvergessen — APPROVED** [review]

**Review by joshtrichards — APPROVED** [review]
:+1:

**Pull request merged**